### ISSUE TITLE
Authorisation-failure- 404. -redirect url

### ISSUE THREAD

**Issue by Senthil-9111** [hmrc/vat-api#587]
Hi there

When granting authority in the production environment of HMRC, I am getting 404 for redirect URL.

This happens after the user clicks grant authority.

In the test scenario it works well, in the production environment I am getting this issue.

I had previously used the live environment and the authorisation worked perfectly.

Any one faced such a problem.

We had used the same production credentials and tokens and others for 2 application connected to the same production environment given by HMRC. 

It worked well for the first one, the second setup is not working fine.

Any ideas would be useful.

**Comment by Senthil-9111** [comment]
This happens after the user enters the HMRC login credentials and accesses their profile and gives grant authority.

Any help from other developers or HMRC will be helpful.

**Comment by JeeBeePee** [comment]
You own the redirect

**Comment by Seren69** [comment]
Hi,  I was getting a similar problem.  Authorisation was working successfully on test but when anyone tried to authorise in production they were getting redirect errors.  I found that the redirect URI I had set up when registering the application in sandbox hadn't been carried over to production.  When I completed the URI against the production application in the Developer Hub authorisation worked correctly.
